src/appleseed.studio/mainwindow/rendering/cameracontroller.h

```diff
@@ -32,6 +32,7 @@
 
 // appleseed.renderer headers.
 #include "renderer/api/rendering.h"
+#include "renderer/kernel/rendering/scenepicker.h"
```

Comment:
In that case, I would recommend introducing `renderer/api/scenepicker.h` instead. Otherwise the rule that only `renderer/api/` headers can be included in client applications is no longer always true and things get confusing.

An alternative would be to keep `scenepicker.h` in `renderer/api/rendering.h` and try to move as much things as possible to the cpp file, but this may not be possible.